src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -215,6 +227,50 @@ protected void interrupted() {
         };
     }
 
+    public Command LowerLiftToLimitSwitch() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+                disengageBrake();
+            }
+
+            @Override
+            protected void execute() {
+                lowerLift();
+                System.out.println(bottomLimitSwitch);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                if (bottomLimitSwitch.get()) {
+                    return false;
+                } else {
+                    return true;
+                }
+            }
+        };
+    }
+
+    public Command LowerLiftForTime() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+                disengageBrake();
+                setTimeout(.1);
```

Comment:
Shouldn't this "0.1" be a parameter going into this function? That way we can change how long it runs every time we call it? Should be something like `public Command LowerLiftforTime(double duration)`